app/models/gravatar.rb

```diff
@@ -0,0 +1,46 @@
+class Gravatar
```

Comment:
Would this fit better as a [Rails Concern](https://api.rubyonrails.org/v7.0.4/classes/ActiveSupport/Concern.html) instead? I feel this would better encapsulate this functionality and remove the depends on a `User` object. May also even reduce `app/helpers/application_helper.rb` I think?